project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.9
+sbt.version=1.3.13
```

Comment:
updated this as the sbt-dependency-graph plugin has issues which are more fully fixed in this version, and snyk relies on that plugin